Magento_Theme/templates/header/links/account.phtml

```diff
@@ -3,6 +3,7 @@ $_href = $block->getHref();
 ?>
 
 <a class="header-button"
```

Comment:
Formatting, move class to new line here and in `contact.phtml`, `wishlist.phtml`. 
Long time ago we didn't have this formatting rule so we should update the code now by the way